src/tests/ftest/util/server_utils_params.py

```diff
@@ -436,6 +436,7 @@ class EngineYamlParameters(YamlParameters):
             "D_LOG_FILE_APPEND_PID=1",
             "DAOS_POOL_RF=4",
             "CRT_EVENT_DELAY=1",
+            "DAOS_VOS_AGG_GAP=25",
```

Comment:
> 1. Why change this from the default of 60?
> 2. I would think we need to run this PR with affected tests? Maybe `Features: aggregation`?

The environment variable `DAOS_VOS_AGG_GAP` is new introduced via this patch. Before that, the gap between aggregation and current HLC was `DAOS_AGG_THRESHOLD`, that was about 20 seconds. Such gap cannot match the new requirement for stable epoch in this patch because it was fixed and may be too short as to cause some transactions to be restarted frequently.
Generally, a small nvme device is more easy to be filled, so need more small aggregation gap, that is usually for test environment, such as for CI test. But in real product environment, we can use relative large configuration, that is why we say the fault is 60 seconds.

VOS aggregation will automatically run at background during CI tests except it is disabled explicitly.

Reply:
My main concern is the default `pr` tests might not be sufficient to cover this change. Even though aggregation is enabled by default, most tests do not explicitly verify aggregation behavior. I don't know this code well enough to say whether or not `pr` is enough.
I don't want to block this PR unnecessarily so I will remove my -1, but please keep in mind that in general it is much more expensive to fix regressions after initial landing than in the original PR, where all we have to do is use `Features: <tags>`

Reply:
I have some concerns about setting this flag for all tests. CI testing should resemble real world testing on functional HW. the per SSD capacity in CI functional tests is not different from production systems, where also the capacity varies greatly depending on what customers want.
It's fine to change it for some particular test, but (correct me if im wrong) this change here is for all CI tests?
If that is the case, we cannot land that TBH.

Reply:
Yeah, this is for all functional CI tests

Reply:
"DAOS_VOS_AGG_GAP=25" this setting makes CI tests almost keep the same behavior as without the patch (was 20+ seconds before). If without such setting, then for some short-time test, VOS aggregation may not be triggered before the test completed, that will decrease the potential race windows with VOS aggregation. Then may hide potential bugs. We can remove such setting for CI tests, but I am afraid that:
1) Space pressure for small pool configuration.
2) Less race with VOS aggregation.